src/utils.ts

```diff
@@ -0,0 +1,41 @@
+import CID from 'cids'
+import isCircular from 'is-circular'
+import transform from 'lodash.transform'
```

Comment:
Seems like this `transform()` function could simply be replaced by `Object.entries().reduce()` rather than adding an extra dependency, ex:
```js
function transform(src, cb) {
  return Object.entries(src).reduce((acc, [key, value]) => {
    cb(acc, value, key);
    return acc;
  }, {});
}
```

Reply:
Good point. I copied the code from here and not sure if there is an explicit reason. https://github.com/ipld/js-dag-json/blob/master/index.js Can probably be changed!